<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addInfo</class>
 <widget class="QDialog" name="addInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="firstLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>201</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>                          First name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="surnameLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>201</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>                             Surname:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="surnameLine">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>331</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="firstLine">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>110</y>
     <width>331</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="holidayLine">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>310</y>
     <width>331</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="vehicleLine">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>350</y>
     <width>331</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="mobileLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>270</y>
     <width>201</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>                    Mobile number:</string>
   </property>
  </widget>
  <widget class="QLabel" name="holidayLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>310</y>
     <width>201</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>                 Holiday duration:</string>
   </property>
  </widget>
  <widget class="QLabel" name="vehicleLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>350</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>                      Number plate:</string>
   </property>
  </widget>
  <widget class="QLabel" name="customerLabel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>30</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Customer Profile</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mobileLine">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>270</y>
     <width>331</width>
     <height>27</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="maxLength">
    <number>11</number>
   </property>
  </widget>
  <widget class="QPushButton" name="submitButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>420</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>420</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>420</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="addressLine">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>190</y>
     <width>331</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="PostcodeLine">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>230</y>
     <width>331</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>8</number>
   </property>
  </widget>
  <widget class="QLabel" name="surnameLabel_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>201</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>                              Address:</string>
   </property>
  </widget>
  <widget class="QLabel" name="surnameLabel_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>230</y>
     <width>201</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>                            Postcode:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="parkCombo">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>80</y>
     <width>31</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>A</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>B</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>C</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="carParkLabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>   Carpark:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="parkEdit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>80</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>4</number>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>firstLine</tabstop>
  <tabstop>surnameLine</tabstop>
  <tabstop>addressLine</tabstop>
  <tabstop>PostcodeLine</tabstop>
  <tabstop>mobileLine</tabstop>
  <tabstop>holidayLine</tabstop>
  <tabstop>vehicleLine</tabstop>
  <tabstop>clearButton</tabstop>
  <tabstop>submitButton</tabstop>
  <tabstop>exitButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>submitButton</sender>
   <signal>clicked()</signal>
   <receiver>addInfo</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>403</x>
     <y>435</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exitButton</sender>
   <signal>clicked()</signal>
   <receiver>addInfo</receiver>
   <slot>exitapp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>532</x>
     <y>436</y>
    </hint>
    <hint type="destinationlabel">
     <x>361</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mobileLine</sender>
   <signal>textEdited(QString)</signal>
   <receiver>addInfo</receiver>
   <slot>number()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>570</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>595</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>addInfo</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>428</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>391</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>firstLine</sender>
   <signal>textEdited(QString)</signal>
   <receiver>addInfo</receiver>
   <slot>letter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>544</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>620</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>surnameLine</sender>
   <signal>textEdited(QString)</signal>
   <receiver>addInfo</receiver>
   <slot>letter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>547</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>648</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>parkCombo</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>addInfo</receiver>
   <slot>carPark()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>exitapp()</slot>
  <slot>number()</slot>
  <slot>clear()</slot>
  <slot>letter()</slot>
  <slot>carPark()</slot>
 </slots>
</ui>
